packages/ui-components/src/components/alert/alert.types.ts

```diff
@@ -0,0 +1,37 @@
+import { EventEmitter } from '@stencil/core';
+import { EComponentSize, EIconName } from '../../types';
+
+export enum EAlertType {
+	Info = 'info',
+	Error = 'error',
+	Success = 'success',
+	Warning = 'warning'
+}
+
+export interface IAlertConfig {
+	/** (required) Defines the type of alert to show, possible values are defined in `EAlertType` */
+	type: EAlertType;
+	/** (optional) Defines the size of the component, defaults to EComponentType.Large */
+	size: EComponentSize;
```

Comment:
you need to add the question mark after all the required fields otherwise they will all be required.

Reply:
👀